kvmagent/kvmagent/test/test_host.py

```
@@ -13,7 +13,7 @@
 
 
 class TestHost(TestCase, pytest_utils.PytestExtension):
-    @misc.test_for(handlers=[
+    @env.test_for(handlers=[
```

Ruff (0.11.9)

`env` may be undefined, or defined from star imports

(F405)